web/src/timerender.ts

```diff
@@ -352,6 +389,21 @@ export function get_markdown_time_tooltip(reference: HTMLElement): DocumentFragm
     return "";
 }
 
+function update_recent_topics_timestamps(): void {
+    for (const entry of recent_topics_update_list) {
+        const conversation_key = entry.conversation_key!;
+        const escaped_selector = $.escapeSelector(":" + conversation_key);
+        const $element = $(`#recent_conversation${escaped_selector} .last_msg_time`);
```

Comment:
We should just use `CSS.escape` for the whole `recent_conversation${escaped_selector} .last_msg_time`.

Reply:
Then it will escape the space between `recent_conversation:[id]` and `.last_msg_time`, which is incorrect. I made changes using CSS.escape. Please let me know if you still have any concerns.